Functions.Templates/Templates-v2/BlobTrigger-JavaScript/index.js

```diff
@@ -0,0 +1,9 @@
+const { app } = require('@azure/functions');

```

Comment:
Should we also add these `.md` files, as in the docs?

Reply:
Are you talking about the `.md` files that the Python team added? Let me know if you feel strongly, but I'm not a fan. First, they feel like an annoying pop-up when used in VS Code and AFAIK there's no way to do "Don't show again". Second, I think they're liable to get out-of-date and I would prefer if the official microsoft docs are the single source of truth. Last, my hope is that our template experience is good enough that users shouldn't need a page of docs just to try stuff out